rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+// rules-tests/Php74/Rector/Property/TypedPropertyRector/Fixture/michael_test.php.inc
+
```

Comment:
This line is compared to half after `-----`, so it should be removed as it's missing in 2nd half.

Reply:
This line is still breakin the test